stingray/largememory.py

```diff
@@ -575,6 +575,7 @@ def retreiveData(data_type, dir_name, path=os.getcwd(), chunk_data=False, chunk_
         raise ValueError((f"Invalid input data: {data_type}"))
 
 
+# REVIEW: Review computation performed
```

Comment:
Could you please review if the averaging and summing performed in `_combineSpectra` and `_addSpectra` is logically correct?

Also, if any more params or attributes need to be summed, averaged(eg. gtis??)